packages/datadog-instrumentations/src/couchbase.js

```diff
@@ -160,60 +161,63 @@ function wrapV3Query (query) {
 }
 
 // semver >=2 <3
-addHook({ name: 'couchbase', file: 'lib/bucket.js', versions: ['^2.6.5'] }, Bucket => {
-  const startCh = channel('apm:couchbase:query:start')
-  const finishCh = channel('apm:couchbase:query:finish')
-  const errorCh = channel('apm:couchbase:query:error')
+if (!semver.satisfies(process.version, '>=14.0.0')) {
```

Comment:
Can this be done in the test instead?

Reply:
I added this here because, when running tests in the `couchbase` job (v3 and higher), using Node 14, it would still try and build v2 using that Node version as well, which would cause a crash (and, trying to run v3 and higher tests with Node 12, which is what works with couchbase v2, would also cause *it* to crash). So, I put this here to not even build couchbase v2 when using Node 14 and higher. If there is a way I can add this to the tests instead, I will!

Reply:
Seems to me like it'd be valuable to just add a `nodeVersions` option to the `addHook(options, ...)` input so checks like this could be done anywhere.

Reply:
This is something I'd be happy to add in a separate PR. It would definitely help in this case too in cleaning up the tests.

Reply:
> I added this here because, when running tests in the couchbase job (v3 and higher), using Node 14, it would still try and build v2 using that Node version as well

This is basically my point, if this code is only for tests it should be in tests. If someone gets couchbase to build on Node 14 for example, there is no reason why our instrumentation shouldn't work. An option could be added to `externals.json` for example.